module/VuFind/src/VuFind/Db/Entity/AccessToken.php

```diff
@@ -38,54 +38,71 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="access_token")
+ * @ORM\Table(name="access_token",
+ * indexes={@ORM\Index(name="user_id", columns={"user_id"})}
+ * )
  * @ORM\Entity
  */
 class AccessToken implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var string
      *
-     * @ORM\Column(name="id", type="string", length=255, nullable=false)
+     * @ORM\Column(type="string", length=255, nullable=false)
      * @ORM\Id
-     * @ORM\GeneratedValue(strategy="NONE")
```

Comment:
Since strategy = NONE is the same as not having GeneratedValue at all, I don't think it hurts to remove this. But might there be any possible advantages to keeping the GeneratedValue annotation, since this is an externally-generated value? (I don't have strong opinions about this, but just want to understand your decision-making here).